mqtt-v5-broker/src/main.rs

```diff
@@ -196,6 +199,12 @@ async fn websocket_server_loop(broker_tx: Sender<BrokerMessage>) {
 }
 
 fn main() -> Result<(), Box<dyn std::error::Error>> {
+    if env::var("RUST_LOG").is_err() {
+        env_logger::builder().filter(None, log::LevelFilter::Debug).init();
+    } else {
+        env_logger::init();
+    }
```

Comment:
Small request, but could you place this within a function named something like `init_logging` or `setup_logging`?

Reply:
Sure. Done.

Reply:
Perfect, thank you!